assessments/projects/opa/README.md

```diff
@@ -0,0 +1,111 @@
+# OPA Security Assessment
+
+Completed: September 2019
+
+Security reviewers: Justin Cappos, Justin Cormack, Brandon Lum, Sarah Allen
+
+Project security lead: Ash Narkar
+
+* Source code: [github.com/open-policy-agent/opa](https://github.com/open-policy-agent/opa)
+* Web site: [openpolicyagent.org](https://www.openpolicyagent.org/)
+
+## Background
+
+Deployed cloud native applications have policies that must be correctly enforced
+for security reasons.  The desired overall policy may span many different pieces
+of software and configurations.  Open Policy Agent (OPA) seeks to consolidate
+and control policy configuration in a consistent manner across the different
+components of a cloud native deployment.
+
+_Maturity_
+
+The core technology is in production with a broad set of companies, including
+Netflix, Cloudflare, Chef, and others ([ADOPTERS.md](https://github.com/open-policy-agent/opa/blob/master/ADOPTERS.md)).
+Code contributions are primarily from Styra, yet there has been community participation
+from a wide range of adopters ([77 contributors](https://github.com/open-policy-agent/opa/graphs/contributors),
+with Chef engineer in top 4 and Google engineer co-authoring RegoV2 specification).
+
+## Summary
```

Comment:
probably an aside to this PR but it would be interesting to plumb deeper into the notion of solving for X vs increased surface area.  Is there a theoretical underpinning that requires that solving X always requires an increase in complexity or surface area?  If complexity or surface area does go up (whether or not it MUST), is that necessarily a bad thing if compensating controls are in place?  ie how to quantify the "breakeven" point on increased risk vs solved risk. anyway, something for another thread to be sure...